libs/forcetk.mobilesdk.js

```diff
@@ -73,13 +73,30 @@ if (forcetk.Client === undefined) {
      * @constructor
      */
     forcetk.Client = function(clientId, loginUrl, proxyUrl, authCallback) {
+        forcetk.Client(clientId, loginUrl, proxyUrl, null, true);
+    }
+
+    /**
+     * The Client provides a convenient wrapper for the Force.com REST API,
+     * allowing JavaScript in Visualforce pages to use the API via the Ajax
+     * Proxy.
+     * @param [clientId=null] 'Consumer Key' in the Remote Access app settings
+     * @param [loginUrl='https://login.salesforce.com/'] Login endpoint
+     * @param [proxyUrl=null] Proxy URL. Omit if running on Visualforce or
+     *                  Cordova etc
+     * @param authCallback Callback method to perform authentication when 401 is received.
+     * @param useNativeNetworking True - if native network stack should be used, False - otherwise.
+     * @constructor
+     */
+    forcetk.Client = function(clientId, loginUrl, proxyUrl, authCallback, useNativeNetworking) {
```

Comment:
Or if you want default on, then the parameter should be called useXhrNetworking (typically undefined is treated like false).